### ISSUE TITLE
0.5.19 breaks jest source maps

### ISSUE THREAD

**Issue by schw4rzlicht** [evanw/node-source-map-support#279]
Hey there!

I'm pretty new to TypeScript/jest and I spent hours in figuring out why line numbering in failing tests are wrong so please bear with me.

I'm not sure if I missed something or it's an actual bug and I couldn't find anything close to my problem online. When running `install()`, line numbering breaks, when commenting it out, everything works fine (but of course stack traces don't work any more). It doesn't matter if `install()` is run in test code or in imported production code, test line numbering always breaks.

I have set up a demo repo over [here](https://github.com/schw4rzlicht/node-sourcemap-jest-demo).

Thanks for your help!

**Comment by schw4rzlicht** [comment]
I just tested 0.5.0 and 0.5.18 and they both have the same problem.

**Comment by andinoJamesV** [comment]
Possibly related and probably not this Libs issue https://github.com/istanbuljs/nyc/issues/1323

**Comment by andinoJamesV** [comment]
So I kept digging into the project and code coverage is built into V8

**Comment by schw4rzlicht** [comment]
@Heavy-Gray I'm not sure how I could actually help.

> My suspicion is mix and match CJS and ES module types are the problem?

Where does that suspicion come from? Did you have a look at the sample I linked above? Do you mean b/c of

```json
{
  "compilerOptions": {
    "esModuleInterop": true
  }
}
```

https://github.com/schw4rzlicht/node-sourcemap-jest-demo/blob/master/tsconfig.json

**Comment by Heavy-Gray** [comment]
@schw4rzlicht No for sure you can`t I was bouncing around broken libraries and by chance this lib and thread came up.

Just to verify node-source-map-support is working and the problem you are having is in nyc or Jest

**Comment by schw4rzlicht** [comment]
> Just to verify node-source-map-support is working and the problem you are having is in nyc or Jest

Yes. Debugging and stack traces have the right line numbers outside of Jest tests.

**Comment by Marik-D** [comment]
Related issue: https://github.com/facebook/jest/issues/10330

**Comment by stavalfi** [comment]
as a workaround:

```
if (!process.env.TEST_MODE) {
  require('source-map-support').install()
}
```

until this issue is fixed.

also, https://github.com/evanw/esbuild has impact as well because of this issue. probably https://github.com/swc-project/swc also.

**Comment by silasdavis** [comment]
Are you using `ts-jest`? I am seeing this issue when using `ts-jest` and I have opened a ticket with minimal repro here: https://github.com/kulshekhar/ts-jest/issues/2372
